### PR TITLE
GitOps release note entry and TP updates

### PR DESCRIPTION
**Pull request by Preeticp**
This PR adds a small note for GitOps in the new features and adds it to the TP table. It also adds TP status to Pipelines 4.7 in the TP table.

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -1026,7 +1028,12 @@ In the table below, features are marked with the following statuses:
 |OpenShift Pipelines
 |TP
 |TP
-|
+|TP
+
+|OpenShift GitOps
+|-
+|-
```
Comment (amitkrout) [inline release_notes/ocp-4-7-release-notes.adoc:1035]: TP

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -1026,7 +1028,12 @@ In the table below, features are marked with the following statuses:
...
+|TP
```
Comment (amitkrout) [inline release_notes/ocp-4-7-release-notes.adoc:1036]: ```GA```. As we are not sync with OpenShift release, our GA will come after 4.7 is released.

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -658,7 +658,9 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-7-dev-exp"]
 === Developer experience
 
-
+==== Red Hat Openshift GitOps (Technology Preview)
+The Red Hat Openshift GitOps 1.0 Technology Preview release introduces a declarative way to implement continuous deployment for cloud native applications. You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and automating secure, predictable, traceable, and repeatable application delivery across hybrid multi-cluster Kubernetes environments.
+It uses Argo CD as the core, and adds other tooling to enable teams to implement GitOps workflows across clusters.
```
Comment (amitkrout) [inline release_notes/ocp-4-7-release-notes.adoc:663]: Looks good to me, however i request somebody from dev team please have a look. 

Ping @wtam2018 @bigkevmcd @chetan-rns

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (amitkrout) [reply release_notes/ocp-4-7-release-notes.adoc:1036]: After discussion over slack with @Preeticp got to know that until the GA is out we can not mark it as GA. Lets go this field as TP.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -658,7 +658,9 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-7-dev-exp"]
 === Developer experience
 
-
+==== Red Hat Openshift GitOps (Technology Preview)
```
Comment (chetan-rns) [inline release_notes/ocp-4-7-release-notes.adoc:661]:
```suggestion
==== Red Hat OpenShift GitOps (Technology Preview)
```
Nit

File: release_notes/ocp-4-7-release-notes.adoc [hunk 5]
```diff
@@ -658,7 +658,9 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
...
+The Red Hat Openshift GitOps 1.0 Technology Preview release introduces a declarative way to implement continuous deployment for cloud native applications. You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and automating secure, predictable, traceable, and repeatable application delivery across hybrid multi-cluster Kubernetes environments.
+It uses Argo CD as the core, and adds other tooling to enable teams to implement GitOps workflows across clusters.  For more information, see, xref:../cicd/gitops/understanding-openshift-gitops.html[Understanding OpenShift GitOps].
```
Comment (chetan-rns) [inline release_notes/ocp-4-7-release-notes.adoc:663]:
```suggestion
It uses Argo CD as the core, and adds other tooling to enable teams to implement GitOps workflows across clusters. For more information, see, xref:../cicd/gitops/understanding-openshift-gitops.html[Understanding OpenShift GitOps].
```
Small nit. Everything else looks good to me!

Comment (Preeticp) [reply release_notes/ocp-4-7-release-notes.adoc:663]: Thank you, Chetan. Will fix it!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 4 above)
Comment (Preeticp) [reply release_notes/ocp-4-7-release-notes.adoc:661]: Good catch, TY!!

**Comment by chetan-rns** [comment]
Looks good to me!
/lgtm

**Review by amitkrout — APPROVED** [review]
LGTM

File: release_notes/ocp-4-7-release-notes.adoc [hunk 6]
```diff
@@ -658,7 +658,9 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-7-dev-exp"]
 === Developer experience
 
-
+==== Red Hat OpenShift GitOps (Technology Preview)
+The Red Hat OpenShift GitOps 1.0 Technology Preview release introduces a declarative way to implement continuous deployment for cloud native applications. You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and automating secure, predictable, traceable, and repeatable application delivery across hybrid multi-cluster Kubernetes environments.
+It uses Argo CD as the core, and adds other tooling to enable teams to implement GitOps workflows across clusters. For more information, see, xref:../cicd/gitops/understanding-openshift-gitops.html[Understanding OpenShift GitOps].
```
Comment (amitkrout) [inline release_notes/ocp-4-7-release-notes.adoc:663]: link does not work

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-7-release-notes.adoc (hunk 6 above)
Comment (amitkrout) [reply release_notes/ocp-4-7-release-notes.adoc:663]: @Preeticp Can please check it in the preview page

Comment (Preeticp) [reply release_notes/ocp-4-7-release-notes.adoc:663]: Ack, trying to debug.

Comment (Preeticp) [reply release_notes/ocp-4-7-release-notes.adoc]: Works, now, updated link a well as rebased my feature branch

File: release_notes/ocp-4-7-release-notes.adoc [hunk 7]
```diff
@@ -732,7 +732,9 @@ For more information, see xref:../scalability_and_performance/cnf-performance-ad
 [id="ocp-4-7-dev-exp"]
 === Developer experience
 
-
+==== Red Hat OpenShift GitOps (Technology Preview)
+The Red Hat OpenShift GitOps 1.0 Technology Preview release introduces a declarative way to implement continuous deployment for cloud native applications. You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and automating secure, predictable, traceable, and repeatable application delivery across hybrid, multi-cluster, Kubernetes environments.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:736]: Something about the intro to the second clause is confusing - do you mean:
`You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and to automate secure, predictable, traceable, and repeatable application delivery across hybrid, multi-cluster, Kubernetes environments.`
or
`You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and for automating secure, predictable, traceable, and repeatable application delivery across hybrid, multi-cluster, Kubernetes environments.`

Comment (Preeticp) [reply release_notes/ocp-4-7-release-notes.adoc:736]: I would say the second, let me make the change and squash the commits

**Review by jeana-redhat — APPROVED** [review]
approving exception

**Pull request merged**